ArticleTemplates/assets/scss/modules/_quiz.scss

```diff
@@ -424,5 +440,72 @@
                 }
             }
         }
+
+        // Quiz Results
+        .quiz-results {
+            .quiz-results {
+                &__description {
+                    color: color(tone-news);
+                    font: 600 18px/1.2 $egyptian-display;
+                }
+
+                &__title {
+                    color: color(tone-news-accent);
+                    font: 100 50px/55px $egyptian-display;
+                }
+            }
+        }
+    
+        // Personality quiz overrides
+        &.personality-quiz {
+            .question__answer {
+                &:not(.has-image) {
+                    .answer__text {
+                        color: color(shade-1);
+                    }
+
+                    .answer__marker {
+                        border: 1px solid color(shade-1);
+
+                        &::before {
+                            content: '';
+                        }
+
+                        .answer__marker__inner {
+                            opacity: 0;
```

Comment:
Yep, I often end up nesting classes at the same level the corresponding node is nested in the dom, which is not really necessary.